### PR TITLE
Create or update baselines to match 2.0

### PR DESCRIPTION
**Pull request by ryanbrandenburg**

### CODE REVIEW

File: src/Microsoft.AspNetCore.Buffering/baseline.netcore.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
+{
+  "AssemblyIdentity": "Microsoft.AspNetCore.Buffering, Version=2.0.2.0, Culture=neutral, PublicKeyToken=adb9793829ddae60",
```
Comment (Tratcher) [inline src/Microsoft.AspNetCore.Buffering/baseline.netcore.json]: ?? This is a prototype, there is no version 2.0.2.

Comment (Eilon) [reply src/Microsoft.AspNetCore.Buffering/baseline.netcore.json]: Hmm yes that's an odd version #...

Comment (Eilon) [reply src/Microsoft.AspNetCore.Buffering/baseline.netcore.json]: As in, it should be 0.4.0 or similar...

Comment (Tratcher) [reply src/Microsoft.AspNetCore.Buffering/baseline.netcore.json]: Can we remove baselines for prototypes?

Comment (Eilon) [reply src/Microsoft.AspNetCore.Buffering/baseline.netcore.json]: @Tratcher yeah good point I think that is the right thing to do. There can't be a breaking change in a pre-1.0.0 package.

**Comment by ryanbrandenburg** [comment]
:up::date:

**Pull request merged**